bundles/AdminBundle/Resources/public/js/pimcore/document/editables/date.js

```diff
@@ -38,8 +38,6 @@ pimcore.document.editables.date = Class.create(pimcore.document.editable, {
             this.config.value = this.data;
         }
 
-        this.config.name = id + "_editable";
```

Comment:
This was called in the wrong place, because `id` isn't defined here. So we better do this in `initialize`.